infrastructure/src/main/java/org/corfudb/infrastructure/orchestrator/Orchestrator.java

```diff
@@ -0,0 +1,78 @@
+package org.corfudb.infrastructure.orchestrator;
+
+import io.netty.channel.ChannelHandlerContext;
+import lombok.extern.slf4j.Slf4j;
+import org.corfudb.infrastructure.IServerRouter;
+import org.corfudb.infrastructure.orchestrator.workflows.AddNode;
+import org.corfudb.protocols.wireprotocol.CorfuMsgType;
+import org.corfudb.protocols.wireprotocol.CorfuPayloadMsg;
+import org.corfudb.protocols.wireprotocol.orchestrator.AddNodeRequest;
+import org.corfudb.protocols.wireprotocol.orchestrator.OrchestratorRequest;
+import org.corfudb.runtime.CorfuRuntime;
+
+import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * Orchestrator is a service that acts on executing actions that belong to workflow instances.
+ * <p>
+ * Created by Maithem on 10/25/17.
+ */
+
+@Slf4j
+public class Orchestrator {
+
+    final CorfuRuntime runtime;
+
+    /**
+     * The id of this management server.
```

Comment:
Does not seem like the id for a management server.

Reply:
Its a "stub" for now. We don't have id generation for nodes yet.

Reply:
Fix the javadoc.

Reply:
removed it for now.